pkg/actions/service.go

```diff
@@ -132,20 +138,32 @@ func protoFromTaskResult(m *TaskResult) *TaskResultData {
 	}
 }
 
-func getTasksPath(repoID, runID string) string {
-	return kv.FormatPath(actionsPrefix, repoID, taskPrefix, runID)
+func GetBaseActionsPath(repoID string) string {
```

Comment:
Sorry for the late comment. Basically the go convention is:
```suggestion
func BaseActionsPath(repoID string) string {
```

Go hates `Get`/`Set` prefixes.